langchain/llms/mlflow.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+from functools import lru_cache
+from typing import Any, List, Dict, Mapping, Optional
+from langchain.callbacks.manager import CallbackManagerForLLMRun
+from langchain.llms.base import LLM
+from pydantic import BaseModel, Extra
+from typing import TYPE_CHECKING
+
+
+if TYPE_CHECKING:
+    import mlflow.gateway
+
+
+@lru_cache()
+def _get_client(gateway_uri: str) -> mlflow.gateway.MlflowGatewayClient:
+    import mlflow.gateway
+
+    return mlflow.gateway.MlflowGatewayClient(gateway_uri)
+
+
+class Params(BaseModel, extra=Extra.allow):
+    temperature: float = 0.0
+    stop: List[str] | None = None
+    max_tokens: Optional[int] = None
+    candidate_count: int = 5
+
+
+class MlflowGateway(LLM):
+    gateway_uri: str
+    route: str
+    params: Params | None = None
```

Comment:
@dbczumar Added `params` which validates the common params and also accepts extra params.